src/bundle/SystemInfo/Value/RepositoryMetrics.php

```diff
@@ -42,7 +42,7 @@ class RepositoryMetrics extends ValueObject implements SystemInfo
     public $versionsCount;
 
     /**
-     * Count of Content Types.
+     * Count of content types.
      *
      * @var int
      */
```

Comment:
> This whole docblock doesn't really bring any value, so I would suggest removing it entirely.

Definitely - to be removed. Just `/** @var int */` needs to be kept and only because I want to avoid QA on `public int $contentTypeCount` which would be the most correct ;)